hugegraph-cassandra/src/main/java/com/baidu/hugegraph/backend/store/cassandra/CassandraStore.java

```diff
@@ -279,13 +275,27 @@ private void mutate(CassandraSessionPool.Session session,
             case ELIMINATE:
                 // Eliminate entry
                 if (entry.selfChanged()) {
-                    this.table(entry.type()).eliminate(session, entry.row());
+                    table.eliminate(session, entry.row());
                 }
                 // Eliminate sub rows (edges)
                 for (CassandraBackendEntry.Row row : entry.subRows()) {
                     this.table(row.type()).eliminate(session, row);
                 }
                 break;
+            case UPDATE_IF_PRESENT:
+                if (entry.selfChanged()) {
+                    // TODO: forward to master-writer node
```

Comment:
TODO: override updateIfAbsent for cassandra by INSERT xx IF NOT EXISTS
https://docs.datastax.com/en/cql-oss/3.3/cql/cql_reference/cqlInsert.html